mod-circulation/src/main/resources/domain/mod-circulation/features/loans.feature

```diff
@@ -85,70 +85,31 @@ Feature: Loans tests
     And match response.itemId == itemId
 
   Scenario: Get Loans collection
-    * def extInstanceTypeId = call uuid1
-    * def extInstitutionId = call uuid1
-    * def extCampusId = call uuid1
-    * def extLibraryId = call uuid1
-    * def extItemId1 = call uuid1
-    * def extItemId2 = call uuid1
-    * def limit = 5
+    
+    * def limit = random(2000)
+    * def extItemBarcode = random(10000)
+    * def extUserBarcode = random(10000)
 
-    #post items
-    * call read('classpath:domain/mod-circulation/features/util/initData.feature@PostInstance') { extInstanceTypeId: #(extInstanceTypeId) }
+    #post an item
```

Comment:
```suggestion
    # post an item
```

Reply:
and if you return two items --->  # post items

Reply:
ok, I returned two items and fixed